docs/source/coupling_traffic/introduction.rst

```diff
@@ -0,0 +1,149 @@
+============
+Introduction
+============
+
+When analyzing the movement of crowds in urban areas — such as at train stations or during the arrival and departure phases of major events — it is essential to consider traffic-related aspects.
+Factors like the frequency and capacity of trains, the positioning of train doors, and the interaction between cars and pedestrians at intersections significantly influence the dynamics of pedestrians.
+To investigate such scenarios holistically, including both traffic and pedestrian dynamics, basic functionalities of *JuPedSim* have been integrated into *Eclipse SUMO*.
+
+`Eclipse SUMO <https://eclipse.dev/sumo/>`__ is an open-source, multi-modal microscopic simulator that supports both vehicular and pedestrian traffic models.
+However, until recently, it lacked the capability to simulate pedestrians freely in 2D space — this is where *JuPedSim* comes in.
+*JuPedSim* fills the gap in *SUMO*’s pedestrian modeling by enabling the simulation of crowd movement in 2D space.
+This required coupling two fundamentally different simulation concepts and data structures, which had to be carefully aligned.
+The integration utilizes the existing framework and data structures of *SUMO*, fitting *JuPedSim* into its simulation processes and tools.
+Not all features of *JuPedSim* are available in *SUMO*.
+Based on specific research projects and use cases, a subset of functionalities was selected.
+As a result, not all Python functionalities of *JuPedSim* are accessible — only those exposed via the C interface can be used within *SUMO*.
+Therefore, *SUMO-JuPedSim* is not intended as a standalone pe-destrian simulation tool; for more advanced pedestrian-specific configurations, *JuPedSim* alone remains the better choice.
```

Comment:
- not intended to serve as 
- "pe-destrian" -> pedestrian
- "better" -> preferred (slightly more neutral)